src/main/java/blackjack/domain/gamer/GameParticipants.java

```diff
@@ -0,0 +1,65 @@
+package blackjack.domain.gamer;
+
+import blackjack.domain.GameRule;
+import blackjack.domain.result.GameStatistics;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class GameParticipants {
+
+    private final Dealer dealer;
+    private final List<Player> players;
+
+    private GameParticipants(Dealer dealer, List<Player> players) {
+        this.dealer = dealer;
+        this.players = players;
+    }
+
+    public static GameParticipants of(List<Player> players, Dealer dealer) {
+        return new GameParticipants(dealer, players);
+    }
+
+    public void dealInitialCards() {
+        List<GameParticipant> gameParticipants = getGameParticipants();
+
+        for (int i = 0; i < GameRule.INITIAL_DEALING_CARD_COUNT.getValue(); i++) {
+            gameParticipants.forEach(dealer::dealCard);
+        }
+
+        dealer.hideCard();
+    }
+
+    public void executeHitPhase() {
+        players.forEach(participant -> {
+                    while (participant.shouldHit()) {
+                        dealer.dealCard(participant);
```

Comment:
```suggestion
                        participant.drawCard(dealer.dealCard());
```
딜러가 참가자를 넘겨받아 참가자의 행위를 제어하기보단, 카드만 내어주고, 참가자가 카드를 받는 것처럼 표현해보는것은 어떨까요?
단순 제안일뿐 꼭 반영하진 않으셔도 됩니다. ㅎㅎ

Reply:
좋은 제안 감사합니다! 저도 처음 설계할 때 상당히 고민했던 문제라 제 생각을 조금 더 자세히 공유해보겠습니다.

말씀하신 것처럼 participant.drawCard(dealer.dealCard()) 형태로 작성하게 되면,
딜러의 카드 딜링은 단순히 카드머신으로부터 카드를 받아 참가자에게 **전달하는 중계자 역할**로 축소됩니다.

이 부분에 대해 조금 더 깊게 고민해봤습니다.
현실 세계에서는 딜러가 기계로부터 나온 카드를 손에 들었다가 전달하는 모습이 자연스럽지만, 코드상에서는 딜러가 카드를 일시적으로 소유하는 모습을 표현할 필요는 없다고 생각했습니다. 다만 카드 머신을 딜러가 관리하는 편이 게임의 흐름상 직관적이기 때문에 현재 구조가 만들어지게 되었습니다.

그래서 현재 코드와 같이 dealer.dealCard(participant) 방식을 택했습니다.
딜러는 카드 배분의 대상자를 결정하고, (인자로 참여자를 받는 것)
카드를 받아 자신의 패에 넣는 동작(자신의 패를 변경하는 것)은 참여자 스스로 수행하도록 하는 것이 더 객체지향적인 메시지 전달 방식이라 생각했습니다.

즉, 카드 머신은 카드의 생성 및 제공 책임만 갖고,
딜러는 누가 카드를 받을지 결정하는 책임만 갖고,
참여자는 자신의 패를 직접 변경(카드를 받음)하는 책임을 가집니다.

다만 카드 배분 호출의 흐름이 딜러를 시작점으로 하고 있다는 점에서 다소 어색하다고 느끼신 부분은 충분히 공감합니다. 거시적인 관점에서 이를 개선할 경우 훨씬 직관적이고 명확한 구조로 발전할 가능성이 충분하다고 생각합니다.

결론적으로, 위에서 나열한 여러 상황의 트레이드 오프를 고려했을 때, 현재 코드(dealer.dealCard(participant))를 선택했음을 알려드리고 싶었습니다.

좋은 관점을 제안해주심에 다시 한 번 감사합니다!